obs-studio-server/source/nodeobs_audio_encoders.cpp

```diff
@@ -227,6 +227,15 @@ const char* GetAACEncoderForBitrate(int bitrate)
 
 #define INVALID_BITRATE 10000
 
+bool IsMultitrackAudioSupported(const char* format)
+{
+	if (format != nullptr && strcmp(format, "flv") == 0) {
```

Comment:
if the format is null we should be returning false right?

Reply:
Yes, fixed it